Source/Wpf/Prism.Wpf/Services/Dialogs/DialogService.cs

```diff
@@ -29,6 +32,7 @@ public void ShowDialog(string name, IDialogParameters parameters, Action<IDialog
         void ShowDialogInternal(string name, IDialogParameters parameters, Action<IDialogResult> callback, bool isModal)
         {
             IDialogWindow dialogWindow = CreateDialogWindow();
+            RegionManager.SetRegionManager((DependencyObject)dialogWindow, _regionManager);
```

Comment:
This RegionManager will work only the first time. subsequent clicks will throw ArgumentException: Region with the given name is already registered:

Can you make this as _regionManager.CreateRegionManager to get scoped region instead?

Reply:
No, you will need to implement scoped regions to do this

Reply:
I'm actually thinking I should remove this now.

Reply:
For the same reason the base vm was removed?

Reply:
exactly!